task.sh

```diff
@@ -1 +1,38 @@
 #! /bin/bash
+
+# Getting values from env variables
+USER="${DB_USER}"
+PASSWORD="${DB_PASSWORD}"
+
+# Checking variables
+if [ -z "$USER" ]; then
+    echo "Environment variable DB_USER is not set."
+    exit 1
+fi
+
+if [ -z "$PASSWORD" ]; then
+    echo "Environment variable DB_PASSWORD is not set."
+    exit 1
+fi
+
+echo "Variables are present"
+
+# Making full backup for ShopDB data base
+mysqldump -u "$USER" -p "$PASSWORD" --databases ShopDB --result-file=full_backup.sql
```

Comment:
Keep the database name but remove the "--database" flag; everything should work. When you add this flag, the dump adds the line 
`USE DATABESE;`